packages/consumption/src/modules/requests/utility/validateRelationshipAttributesWithinRequest.ts

```diff
@@ -0,0 +1,206 @@
+import {
+    CreateAttributeRequestItem,
+    ProposeAttributeRequestItem,
+    ReadAttributeRequestItem,
+    RelationshipAttribute,
+    RelationshipAttributeQuery,
+    RequestItem,
+    RequestItemGroup
+} from "@nmshd/content";
+import { CoreAddress } from "@nmshd/core-types";
+import { ConsumptionCoreErrors } from "../../../consumption/ConsumptionCoreErrors";
+import { ValidationResult } from "../../common";
+import { DecideRequestItemGroupParametersJSON } from "../incoming/decide/DecideRequestItemGroupParameters";
+import { DecideRequestItemParametersJSON } from "../incoming/decide/DecideRequestItemParameters";
+
+interface RelationshipAttributeFragment {
+    owner: string;
+    key: string;
+    value: { "@type": string };
+}
+
+type ContainsDuplicateRelationshipAttributeFragmentsResponse =
+    | { containsDuplicates: false }
+    | {
+          containsDuplicates: true;
+          duplicateFragment: RelationshipAttributeFragment;
+      };
```

Comment:
```suggestion
type ContainsDuplicateRelationshipAttributeFragmentsResponse =
    { containsDuplicates: false }
    | {
          containsDuplicates: true;
          duplicateFragment: RelationshipAttributeFragment;
      };
```

Reply:
I can't remove it, when I save, it is automatically added again. :D